### PR TITLE
chore: add Chinese translation, wrap some text inside translation function  

### PR DESCRIPTION
**Pull request by grace-shi**
### SUMMARY
1. add Chinese translations
2. Some text are not translated, use `t` function for translation

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2021-06-11 at 2 22 51 PM

### TESTING INSTRUCTIONS
No test required.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 1]
```diff
@@ -2097,8 +2097,8 @@
       "Alert failed": ["警报失败"],
       "Nothing triggered": ["无触发"],
       "Alert Triggered, In Grace Period": ["警报已触发，在宽限期内"],
-      "Email": [""],
-      "Slack": [""],
+      "Email": ["电子邮件"],
+      "Slack": ["懒惰"],
```
Comment (zhaoyongjie) [inline superset/translations/zh/LC_MESSAGES/messages.json:2101]: Slack is a IM software

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 2]
```diff
@@ -2249,7 +2249,9 @@
       "Connection": ["测试连接"],
       "Database name": ["数据库名称"],
       "Name your dataset": ["命名数据集"],
-      "dialect+driver://username:password@host:port/database": [""],
+      "dialect+driver://username:password@host:port/database": [
```
Comment (zhaoyongjie) [inline superset/translations/zh/LC_MESSAGES/messages.json:2252]: driver pattern no translation required

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 3]
```diff
@@ -1377,7 +1377,7 @@
       "Select a database": ["选择一个数据库"],
       "Force refresh schema list": ["强制刷新数据"],
       "Select a schema (%s)": ["选择一个schema（%s）"],
-      "Schema:": [""],
+      "Schema:": ["模式。"],
```
Comment (zhaoyongjie) [inline superset/translations/zh/LC_MESSAGES/messages.json:1380]: `Schema` is a namespace for a specific database.

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 4]
```diff
@@ -442,7 +442,7 @@
       ],
       "Database": ["数据库"],
       "Last Changed": ["更新时间"],
-      "Schema": ["模式"],
+      "Schema": [""],
```
Comment (zhaoyongjie) [inline superset/translations/zh/LC_MESSAGES/messages.json:445]: same after

**Comment by grace-shi** [comment]
Could anyone help me to find out why this test failed? I didn't change any python code, only update frontend code and added some translations.

**Comment by zhaoyongjie** [comment]
> Could anyone help me to find out why this test failed? I didn't change any python code, only update frontend code and added some translations.

The ci may be stuck, I will run it again.

**Comment by zhaoyongjie** [comment]
`superset/translations/zh/LC_MESSAGES/messages.po` looks like there are some problems

https://github.com/apache/superset/pull/15111/checks?check_run_id=2801417938

**Comment by grace-shi** [comment]
> `superset/translations/zh/LC_MESSAGES/messages.po` looks like there are some problems
> 
> https://github.com/apache/superset/pull/15111/checks?check_run_id=2801417938

fixed python lint

**Comment by zhaoyongjie** [comment]
> > `superset/translations/zh/LC_MESSAGES/messages.po` looks like there are some problems
> > https://github.com/apache/superset/pull/15111/checks?check_run_id=2801417938
> 
> fixed python lint

Thanks for the patience. Run CI again

File: superset/translations/zh/LC_MESSAGES/messages.json (hunk 1 above)
Comment (zhaoyongjie) [inline superset/translations/zh/LC_MESSAGES/messages.json:2101]: should we remove "懒惰“  for the translation?

**Comment by villebro** [comment]
@grace-shi thanks for this PR! I'd love to get this merged, but this needs a rebase + did you notice the review comment by Yongjie?

**Pull request closed without merging**